en/docs/how-tos/support-custom-claims-mapping.md

```diff
@@ -0,0 +1,99 @@
+# Support Custom Claims Mapping
+
+When JWT retrieved from Multiple Identity providers, Microgateway can map the relevant claims as a supported one to validates the JWT.  The claims of the incoming authentication JWT can differ based on the Authorization server that issued the JWT token. This feature maps the incoming jwt remote claims to the local claims of the microgateway as well as the claims expected by the back end service.  You can use custom claims mapping transformation in micro gateway when you have different keys or values of claims in your JWT token to make it as a Microgateway supported one.
+
+In a detailed view for an example if you generate JWT with claims with a different key like claim key is ‘scp’ and value is ‘[“write”, “read”]’. But in Microgateway it only validates correctly if the key is scope and values of scope separated by space as a string. Hence through the custom claims mapping feature, you can map the remote claims to local claims to support the Microgateway.  Here remote claims come from an external provider thus local claims need to be mapped.
+ 
+1. If the key of the remote claims is different from the custom claims, then you need to change the configuration as below.  You can define the multiple keys of the remote claim and local claim.
+
+     Adding the following section to the micro-gw.conf file which is located in the <MGW-RUNTIME-HOME>/conf directory, you can change the configurations in the jwtTokenConfig. Here the remote claim is scp and the local claim is scope.
+#### Configuring claims key
+```
+[[jwtTokenConfig]]
+     issuer = "https://localhost:9443/oauth2/token"
+     audience = "http://org.wso2.apimgt/gateway"
+     certificateAlias = "wso2apim310"
+     #support custom claim mapping
+     claimMapperClassName = "org.wso2.micro.gateway.jwtTransformer.DefaultJwtTransformer"
+     [[jwtTokenConfig.claims]]
+       remoteClaim = "scp"
+       localClaim = "scope" 
+```
+         
+      
+2. If the format of the remote claim value differs from the custom claims then you should write a class to transform it into the correct format and define the claim mapping class name in the config file.
+
+     The following subsections explain how you can work with claims value transformation. Here the existing scope value is in an array and it shoud be change to string with space.
```

Comment:
The following subsections explain how you can work with claims value transformation. Here the existing scope value is in an array and it **should be changed** to string with space.